src/main/groovy/wooga/gradle/release/version/semver2/VersionStrategies.groovy

```diff
@@ -169,7 +169,8 @@ final class VersionStrategies {
      */
     static final SemVerStrategy DEVELOPMENT = Strategies.DEVELOPMENT.copyWith(
             normalStrategy: scopes,
-            buildMetadataStrategy: NetflixOssStrategies.BuildMetadata.DEVELOPMENT_METADATA_STRATEGY)
+            buildMetadataStrategy: NetflixOssStrategies.BuildMetadata.DEVELOPMENT_METADATA_STRATEGY,
+            enforcePrecedence: false)
```

Comment:
Why add this as well?

Reply:
A well that seems to be something I was working on.
Let me clean that up